tests/runtime_neutral/test_codex_memory_user_simulation.py

```
@@ -0,0 +1,652 @@
+def install_codex(target_root: Path, *, env: dict[str, str]) -> None:
+    command = [
+        "bash",
+        str(INSTALL_SCRIPT),
+        "--host",
+        "codex",
+        "--profile",
+        "full",
+        "--target-root",
+        str(target_root),
+    ]
+    subprocess.run(
```

Ruff (0.15.9)

[error] `subprocess` call: check for execution of untrusted input

(S603)